docs/testing/stencil-testrunner/02-config.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Testing Config
+sidebar_label: Config
+description: Testing Config
+slug: /testing-config
+---
+
+# Testing Config
+
+The `testing` config setting in `stencil.config.ts` specifies an object that corresponds to the jest configuration that should be used in your tests. Stencil provides a default configuration, which you likely won't need to edit, however it can be extended with the same configuration options as Jest. See the [Configuring Jest Guide](https://jestjs.io/docs/en/configuration.html) for configuration details.
+
+:::note
+Keep in mind that the usual way of configuring Jest (`package.json` and `jest.config.js`) is not used with the `stencil testing` command. Jest can still be used, but configuring the presets, transpilation and setting up the correct commands must be done by the project.
+:::
+
+Some additional Stencil specific options may be set here as well for configuring the e2e tests:
```

Comment:
Non-blocking - I know this is a copy over from the previous file, but can we end this sentence in a period?

```suggestion
Some additional Stencil specific options may be set here as well for configuring the e2e tests.
```

Right now (to me) it reads if the code is a continuation of just e2e tests, when that's not the case.